### ISSUE TITLE
Cannot upgrade to Matomo 4.0.5

### ISSUE THREAD

**Issue by Laul0** [bitnami/bitnami-docker-matomo#66]
I'm using the Docker image. After upgrade the image to the latest version and upgrade my container, this one does not up anymore... I can not access to the bash neither.

Seriously, I will stop to upgrade my instance because each time I have this kind of surprise:

```
rsync: chgrp "/bitnami/matomo/plugins/VisitorInterest" failed: Operation not permitted (1)
rsync: chgrp "/bitnami/matomo/plugins/VisitorInterest/Columns" failed: Operation not permitted (1)
rsync: chgrp "/bitnami/matomo/plugins/VisitorInterest/Reports" failed: Operation not permitted (1)
rsync: chgrp "/bitnami/matomo/plugins/VisitorInterest/config" failed: Operation not permitted (1)
rsync: chgrp "/bitnami/matomo/plugins/VisitorInterest/lang" failed: Operation not permitted (1)
rsync: chgrp "/bitnami/matomo/plugins/VisitsSummary" failed: Operation not permitted (1)
rsync: chgrp "/bitnami/matomo/plugins/VisitsSummary/Reports" failed: Operation not permitted (1)
rsync: chgrp "/bitnami/matomo/plugins/VisitsSummary/Widgets" failed: Operation not permitted (1)
rsync: chgrp "/bitnami/matomo/plugins/VisitsSummary/config" failed: Operation not permitted (1)
rsync: chgrp "/bitnami/matomo/plugins/VisitsSummary/lang" failed: Operation not permitted (1)
rsync: chgrp "/bitnami/matomo/plugins/VisitsSummary/stylesheets" failed: Operation not permitted (1)
rsync: chgrp "/bitnami/matomo/plugins/VisitsSummary/templates" failed: Operation not permitted (1)
rsync: chgrp "/bitnami/matomo/plugins/WebsiteMeasurable" failed: Operation not permitted (1)
rsync: chgrp "/bitnami/matomo/plugins/WebsiteMeasurable/Settings" failed: Operation not permitted (1)
rsync: chgrp "/bitnami/matomo/plugins/WebsiteMeasurable/config" failed: Operation not permitted (1)
rsync: chgrp "/bitnami/matomo/plugins/WebsiteMeasurable/lang" failed: Operation not permitted (1)
rsync: chgrp "/bitnami/matomo/plugins/Widgetize" failed: Operation not permitted (1)
rsync: chgrp "/bitnami/matomo/plugins/Widgetize/angularjs" failed: Operation not permitted (1)
rsync: chgrp "/bitnami/matomo/plugins/Widgetize/angularjs/export-widget" failed: Operation not permitted (1)
rsync: chgrp "/bitnami/matomo/plugins/Widgetize/angularjs/widget-preview" failed: Operation not permitted (1)
rsync: chgrp "/bitnami/matomo/plugins/Widgetize/config" failed: Operation not permitted (1)
rsync: chgrp "/bitnami/matomo/plugins/Widgetize/lang" failed: Operation not permitted (1)
rsync: chgrp "/bitnami/matomo/plugins/Widgetize/stylesheets" failed: Operation not permitted (1)
rsync: chgrp "/bitnami/matomo/plugins/Widgetize/templates" failed: Operation not permitted (1)
rsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1207) [sender=3.1.3]
```

After the first reboot, I had permissions issues, so I tried to overwrite the plugins folder `chmod -R 777 /bitnami/matomo/plugins`. That remove the access denied but now, I don't know what I have to do with _Operation not permitted_

```
rsync: failed to set times on "/bitnami/matomo/plugins/TwoFactorAuth/lang": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/TwoFactorAuth/stylesheets": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/TwoFactorAuth/templates": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserCountry": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserCountry/Categories": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserCountry/Columns": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserCountry/Commands": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserCountry/Diagnostic": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserCountry/LocationProvider": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserCountry/LocationProvider/GeoIp": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserCountry/ProfileSummary": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserCountry/Reports": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserCountry/angularjs": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserCountry/angularjs/location-provider-selection": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserCountry/angularjs/location-provider-updater": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserCountry/config": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserCountry/lang": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserCountry/stylesheets": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserCountry/templates": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserCountryMap": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserCountryMap/Categories": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserCountryMap/Widgets": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserCountryMap/config": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserCountryMap/images": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserCountryMap/javascripts": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserCountryMap/javascripts/vendor": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserCountryMap/lang": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserCountryMap/stylesheets": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserCountryMap/svg": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserCountryMap/templates": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserId": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserId/Categories": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserId/Columns": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserId/Reports": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserId/config": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserId/javascripts": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserId/lang": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserLanguage": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserLanguage/Columns": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserLanguage/Reports": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserLanguage/config": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UserLanguage/lang": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UsersManager": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UsersManager/Sql": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UsersManager/angularjs": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UsersManager/angularjs/capabilities-edit": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UsersManager/angularjs/capability-edit": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UsersManager/angularjs/paged-users-list": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UsersManager/angularjs/permissions-metadata": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UsersManager/angularjs/personal-settings": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UsersManager/angularjs/user-edit-form": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UsersManager/angularjs/user-permissions-edit": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UsersManager/angularjs/users-manager": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UsersManager/config": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UsersManager/images": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UsersManager/lang": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UsersManager/stylesheets": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/UsersManager/templates": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/VisitFrequency": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/VisitFrequency/Columns": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/VisitFrequency/Columns/Metrics": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/VisitFrequency/Reports": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/VisitFrequency/config": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/VisitFrequency/lang": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/VisitTime": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/VisitTime/Categories": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/VisitTime/Columns": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/VisitTime/DataTable": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/VisitTime/DataTable/Filter": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/VisitTime/Reports": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/VisitTime/config": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/VisitTime/lang": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/VisitorInterest": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/VisitorInterest/Columns": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/VisitorInterest/Reports": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/VisitorInterest/config": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/VisitorInterest/lang": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/VisitsSummary": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/VisitsSummary/Reports": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/VisitsSummary/Widgets": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/VisitsSummary/config": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/VisitsSummary/lang": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/VisitsSummary/stylesheets": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/VisitsSummary/templates": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/WebsiteMeasurable": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/WebsiteMeasurable/Settings": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/WebsiteMeasurable/config": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/WebsiteMeasurable/lang": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/Widgetize": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/Widgetize/angularjs": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/Widgetize/angularjs/export-widget": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/Widgetize/angularjs/widget-preview": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/Widgetize/config": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/Widgetize/lang": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/Widgetize/stylesheets": Operation not permitted (1),
rsync: failed to set times on "/bitnami/matomo/plugins/Widgetize/templates": Operation not permitted (1),
rsync: failed to set permissions on "/bitnami/matomo/plugins/Widgetize/templates/iframe.twig": Operation not permitted (1),
rsync: failed to set permissions on "/bitnami/matomo/plugins/Widgetize/templates/iframe_empty.twig": Operation not permitted (1),
rsync: failed to set permissions on "/bitnami/matomo/plugins/Widgetize/templates/index.twig": Operation not permitted (1),
rsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1207) [sender=3.1.3],
```
Any idea ?

**Comment by nguyenviettuananh** [comment]
@Laul0 how did you solve this problem ?

**Comment by Laul0** [comment]
Hi @nguyenviettuananh, I fixed it like this: #67
